packages/aws-cdk/lib/api/deployments/asset-publishing.ts

```diff
@@ -66,67 +56,19 @@ export async function publishAssets(
 
   const publisher = new AssetPublishing(manifest, {
     aws: new PublishingAws(sdk, targetEnv),
-    progressListener: new PublishingProgressListener(options.quiet ?? false),
+    progressListener: new PublishingProgressListener(),
     throwOnError: false,
     publishInParallel: options.parallel ?? true,
-    buildAssets: options.buildAssets ?? true,
+    buildAssets: true,
     publishAssets: true,
-    quiet: options.quiet,
+    quiet: false,
```

Comment:
Only a single internal call-site of `publishAssets` is left and it is not specifying `quiet` or `buildAssets`. So we just use the default value.